proposals/extensions.md

```diff
@@ -313,8 +313,115 @@ However, beyond implementing the language semantics it must satisfy certain requ
 
 These requirements need more refinement as implementation progresses, and may need to be compromised in corner cases in order to allow for a reasonable implementation approach.
```

Comment:
One of the bullet points above says the generated artifacts should be hidden via modreqs but the new specification below doesn't seem to do that.

Reply:
Good question. Not sure where all we should put modreqs. I'll add an open question